manager-bundle/skeleton/public/.htaccess

```diff
@@ -10,6 +10,112 @@
         # Allow CORS on the Contao TinyMCE skin.
         RewriteRule ^assets/tinymce4/js/skins/contao/fonts/ - [ENV=CONTAO_TINYMCE_SKIN:true]
         Header set Access-Control-Allow-Origin "*" env=CONTAO_TINYMCE_SKIN
+
+        # Do not cache source map files
+        <FilesMatch "\.map$">
+            Header set Expires "0"
+            Header set Cache-Control "no-cache, no-store, must-revalidate, max-age=0, proxy-revalidate, no-transform"
+        </FilesMatch>
+    </IfModule>
+
+    <IfModule mod_filter.c>
+        # Brotli compression
+        <IfModule mod_brotli.c>
+            # Force proxies to cache compressed and non-compressed files separately
+            <IfModule mod_headers.c>
+                <FilesMatch ".(appcache|bmp|css|eot|geojson|html|htm|htc|ics|ico|js|json|jsonld|manifest|md|rdf|rss|svg|ttf|txt|vcard|vtt|wasm|webapp|webmanifest|woff2|xdr|xml|xht|xhtml|xloc)$">
+                    Header append Vary "Accept-Encoding"
+                </FilesMatch>
+            </IfModule>
+
```

Comment:
According to the documentation, `mod_brotli` as well as `mod_deflate` send the `Vary: Accept-Encoding` header automatically. So both `Header append Vary` blocks should not be needed.
```suggestion
```

Reply:
Good catch, I have adjusted the code.